modules/gltf/src/lib/extensions/KHR_draco_mesh_compression.js

```diff
@@ -59,7 +59,15 @@ async function decompressPrimitive(primitive, scenegraph, options, context) {
 
   // this will generate an exception if DracoLoader is not installed
   const {parse} = context;
-  const decodedData = await parse(bufferCopy, DracoLoader, options, context);
+  const dracoOptions = {
```

Comment:
```suggestion
  const dracoOptions = {...options};
  // The entire tileset might be included, too expensive to serialize
  delete options['3d-tiles'];
```

More robust if we just delete the problematic part rather than explicitly pass what is needed. I am assuming the tileset prop is in the 3d-tiles suboject.

Reply:
Hi, yes it is in `3d-tiles` but I saw some more data such as `gltf` (some booleans), `log` (console objects), `CDN` (maybe that one is actually needed?), `baseUri` and `fetch.headers`.  Thinking there might be more objects coming up in different use cases that use GLTF I thought to go for the whitelisting approach, that way `GLTFLoader` doesn't need to 'know' about its users and performance of worker threads could be optimized.